lib/client-assets.php

```diff
@@ -442,8 +442,8 @@ function gutenberg_register_scripts_and_styles() {
 			'wp-is-shallow-equal',
 			'wp-keycodes',
 			'wp-polyfill',
-			'wp-url',
 			'wp-rich-text',
+			'wp-url',
```

Comment:
@atimmer @omarreiss @pento  - pinging you, so you aware that this is coming up soon and we will have to update Core, too.

Reply:
> Just a quick note that we should notify the people working on Core integration because this impacts the scripts registration...

@youknowriad good call, done 😄

Reply:
There's also a new script to register...

Reply:
Thanks for the notification, putting it on my list for when I update the packages based on Gutenberg 4.2.